TeachMeTW/WhoIntedMe
# #18 Base DB

## What was changed:
- Added DB

## How to test
- Check out comments I left, add some models
- Check out SQLite viewer
